.circleci/config.yml

```diff
@@ -337,7 +374,7 @@ commands:
       - run:
           name: Run Navigation Core SDK Unit Tests and generate Jacoco test report
           command: |
-            make core-unit-tests-jacoco
+            make core-unit-tests-release-jacoco
```

Comment:
We'd also have to setup nightly builds that run both release and debug variants.

Reply:
Build all variants for main branch